tests/console/check_system_info.pm

```diff
@@ -81,6 +81,20 @@ sub check_buildid {
     }
 }
 
+# SLE-21783: Update nodejs-common for SLE15 SP4
+# check the nodejs16 is releaseed to SLE15 SP3 & SLE15 SP4
+# check the nodejs-common points to nodejs16
+# test steps:
+# 1) install nodejs-common on SLES15SP3 and migrated SLES15SP4 when wsm is available in SCC_ADDONS
+# 2) check nodejs16 is installed together with nodejs-common
+sub check_nodejs_common {
+    record_info('SLE-21783', 'check nodejs-common');
+    zypper_call('in nodejs-common', log => 'zypper.log');
+    die "nodejs-common doesn't point to nodejs16 as default" unless (script_run('grep -E "nodejs16" /tmp/zypper.log') == 0);
```

Comment:
Could any more specific msg can be match to verify this?
nodejs-common will use soft link ot nodejs16?

Reply:
> Could any more specific msg can be match to verify this? nodejs-common will use soft link ot nodejs16?
Below is the feature description, it doesn't mention soft link to nodejs16:
Current nodejs-common points nodejs14 as the default version (nodejs-default). Nodejs16 will be released at end of October and will be included in SP3 and also ship in SP4. nodejs-common package needs to be updated to points nodejs-default
In the before, nodejs14 is the default version, now nodejs16 is the default version.

Reply:
> Could any more specific msg can be match to verify this? nodejs-common will use soft link ot nodejs16?

Below is the feature description, it doesn't mention soft link to nodejs16:
Current nodejs-common points nodejs14 as the default version (nodejs-default). Nodejs16 will be released at end of October and will be included in SP3 and also ship in SP4. nodejs-common package needs to be updated to points nodejs-default
In the before, nodejs14 is the default version, now nodejs16 is the default version.

Reply:
Why not assert_script_run('grep -E "nodejs16" /tmp/zypper.log') ?

Reply:
Thanks for your comments, updated the code, please help to review, thanks.